game/scripts/npc/abilities/legion_commander_duel.txt

```diff
@@ -55,7 +55,7 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "reward_damage"                                   "10 18 26 42 58"
+        "reward_damage"                                   "20 30 40 50 60" //OAA
```

Comment:
don't double level 1 reward

You're underestimating how strong legion can be with these buffs.

I also suggest we cap the duration to 6 because 7 second duel is fucking stupid to be fair.
Duration: 4.5/5/5.5/6/6
Reward damage: 12/24/36/48/60

Honestly you might just want to leave the duration as it is and just chang reward damage to what I put.

Reply:
He is increasing duration because of stupid status resistance.

Reply:
you can just code it to not be affected by status resistance.